README.md

```diff
@@ -181,24 +181,47 @@ A FRAME pallet is compromised of a number of blockchain primitives:
 First, install [Docker](https://docs.docker.com/get-docker/) and
 [Docker Compose](https://docs.docker.com/compose/install/).
 
-Then run the following command to start a single node development chain.
+Please follow the steps given below to run peaq-network-node parachian on your local machine connected with polkadot
+relaychain running in the PEAQ development environment. It is assumed that you have already downloaded the source code
+for peaq-network-node from the git repository
 
+1. CD into the peaq-network-node directory:
 ```bash
-./scripts/docker_run.sh
+cd peaq-network-node
 ```
 
-This command will firstly compile your code, and then start a local development network. You can
-also replace the default command
-(`cargo build --release && ./target/release/peaq-node --dev --ws-external`)
-by appending your own. A few useful ones are as follow.
+2. Checkout the appropriate branch:
+```bash
+git checkout run_parachain_with_docker_commands
```

Comment:
Should we add the prefix or suffix about the `peaq-dev chain` into the branch's name? I guess it improves the readability; what do you think?

Reply:
Yes, I was thinking on the same lines but the branch name will be a little lengthy in that case. Like:

New Suggested  Name : run_peaq_dev_parachain_with_docker_commands. 

For now I am renaming this branch to the New Suggested Name. Let me know if you have a more shorter and meaning full name in your mind.